addon/components/bsy-switch.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+import layout from '../templates/components/bsy-switch';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  layout,
+  theme: "light",
+  size: "small",
+  showLabels: false,
+  onLabel: 'On',
+  offLabel: 'Off',
```

Comment:
Can we not mix single quotes with double quotes?

Reply:
:+1: